api/resources/translations/messages-en.properties

```diff
@@ -576,6 +576,13 @@ contact.type.place.edit = Edit place
 contact.type.place.new = New place
 contact.type.wrong = Wrong type, contact is not a person.
 contact.updated = Contact updated
+duplicate_check.contact.duplication_message = {{count}} duplicate siblings found
+duplicate_check.contact.acknowledge = Acknowledge duplicate siblings and proceed with submission
+duplicate_check.contact.item_number = Item number:
+duplicate_check.contact.name = Name:
+duplicate_check.contact.created_on = Created on:
+duplicate_check.contact.load_summary_info = Load contact summary info
+duplicate_check.contact.go_to = Take me there
```

Comment:
nitpick: A more common wording would probably just be "Open".  I think it would still be clear enough that by selecting that button you would be leaving the current form and going to that contact.